main.go

```diff
@@ -338,17 +324,16 @@ func killProcess(theProcessType ProcessType) error {
 */
 func runPrep(commandString string) (*exec.Cmd, error) {
 	var err error
-	Info.log("Running Install: " + commandString)
+	ControllerInfo.log("Running APPSODY_PREP command: " + commandString)
```

Comment:
I think this log message should be moved down right before `cmd.Run()` is called.